### PR TITLE
#15-Search bar

### PR DESCRIPTION
**Pull request by renatarko**
created search bar and added on MainScreen component - Issue#15

image

### CODE REVIEW

**Review by tgmarinho — APPROVED** [review]

File: src/components/SearchBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <InputLeftElement pointerEvents="none">
+          <MdOutlineSearch color="#34333A" fontSize={22} />
+        </InputLeftElement>
+
+        <Input
+          border={'none'}
+          placeholder="search item"
+          _placeholder={{
+            color: 'rgba(189, 189, 189, 1)',
+            fontSize: 15,
```
Comment (brunocroh) [inline src/components/SearchBar/index.tsx:17]: Change de values to be equal to figma

```suggestion
            color: '#BDBDBD',
            fontSize: 14,
```

File: src/components/SearchBar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          }}
+          borderRadius="12px"
+          bg={'rgba(255, 255, 255, 1)'}
```
Comment (brunocroh) [inline src/components/SearchBar/index.tsx:20]: Change to HEX color please

File: src/components/SearchBar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { Flex, Input, InputGroup, InputLeftElement } from '@chakra-ui/react';
+import { MdOutlineSearch } from 'react-icons/md';
+
+export const SearchBar = () => {
+  return (
+    <Flex m="1.6rem 4rem">
```
Comment (brunocroh) [inline src/components/SearchBar/index.tsx:6]: Change these units to px

**Review by brunocroh — CHANGES_REQUESTED** [review]

**Comment by brunocroh** [comment]
I merged another PR, and a conflict appear, you can resolve these for us?

You did a great job @renatarko!

**Review by brunocroh — APPROVED** [review]

**Comment by brunocroh** [comment]
I wil merge it, thanks @renatarko !

**Pull request merged**